pulsar-client/src/main/java/org/apache/pulsar/client/api/ConsumerBuilder.java

```diff
@@ -94,6 +94,11 @@
      */
     ConsumerBuilder<T> topics(List<String> topicNames);
 
+    /**
+     *
+     */
+    ConsumerBuilder<T> addTopic(String topicName, Schema<T> schema);
```

Comment:
This is a bit different from the the other `topic()` `topics()` methods. 

Also I find it confusing having a per topic schema, when the consumer itself needs to be created with a schema

Reply:
> Multi-subscribe assumes that everything has a single schema (this may be a flawed assumption) 

I think that's an intuitive assumption: 
 * Consumer has a type 
 * It can consume from multiple topics.. as long as data has same format

To consume from different topics, with different types, we should just use different consumer instances.